### PR TITLE
Feature/calendar test

### PR DESCRIPTION
**Pull request by yannhn**

### CODE REVIEW

File: src/components/Calendar/FullCalendar/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import FullCalendar from '@fullcalendar/react'; // must go before plugins
```
Comment (JessicaLoers) [inline src/components/Calendar/FullCalendar/index.js:1]: I think you don't need the comments in `rows 1 - 2`

File: src/components/Calendar/FullCalendar/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+					plugins={[dayGridPlugin, listPlugin, interactionPlugin]}
+					firstDay={1}
+					headerToolbar={{
+						left: 'dayGridMonth',
+						center: 'title',
+						right: 'prev,next,today',
+					}}
+					initialView="dayGridMonth"
+					dateClick={function (info) {
+						this.changeView('dayGridDay', info.dateStr);
```
Comment (JessicaLoers) [inline src/components/Calendar/FullCalendar/index.js:22]: `this.changeView()`
This seems to be the old React syntax, please check and adapt. Anyway, does `dateClick` works and is executed? 🤔

File: src/components/Calendar/FullCalendar/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+					}}
+					events={[
```
Comment (JessicaLoers) [inline src/components/Calendar/FullCalendar/index.js:24]: Will these entries be created dynamically in one of the next user stories?

File: src/components/Calendar/FullCalendar/styled.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const StyleWrapper = styled.section`
```
Comment (JessicaLoers) [inline src/components/Calendar/FullCalendar/styled.js:3]: If you only need this element for styling, i.e. there is no headline or similar in here, you can use a `<div>` here.

**Review by JessicaLoers — COMMENTED** [review]
**Hi @yannhn -  well done ✌️**
Please think about my comments - especially the point about the old React syntax.

File: src/components/Calendar/FullCalendar/index.js (hunk 3 above)
Comment (yannhn) [reply src/components/Calendar/FullCalendar/index.js:24]: Yes!

**Comment by yannhn** [comment]
Thanks for the feedback @JessicaLoers! Please take a look at my comments.

File: src/components/Calendar/FullCalendar/index.js (hunk 2 above)
Comment (yannhn) [reply src/components/Calendar/FullCalendar/index.js:22]: Nevermind, I just fixed it without the dateClick!

**Pull request merged**